2-ui/1-document/04-searching-elements-dom/1-find-elements/task.md

```diff
@@ -2,17 +2,17 @@ importance: 4
 
 ---
 
-# Search for elements
+# جستجوی elementها
 
-Here's the document with the table and form.
+اینجا document مربوط به table و form را می‌بینیم. 
 
-How to find?...
+چگونه پیدا کنیم؟
```

Comment:
```suggestion
چگونه ... را پیدا کنیم؟
```

Reply:
> 

```suggestion
چگونه موارد زیر را پیدا کنیم؟
```
The [ellipsis](https://en.wikipedia.org/wiki/Ellipsis) refers to the follow-up lines.